### ISSUE TITLE
mask=1,2,3 is wrong (should be 0,1,2) look at your yolov3-tiny.cfg

### ISSUE THREAD

**Issue by MaratZakirov** [eriklindernoren/PyTorch-YOLOv3#505]
Compare your CFG with the original from https://github.com/pjreddie/darknet/blob/master/cfg/yolov3-tiny.cfg

**Comment by Flova** [comment]
We use the pre-trained weights from pjreddie and due to an error in the original repo, they need to be trained with that settings. See https://github.com/pjreddie/darknet/issues/851
